metrics/examples/prometheus/alerting.yaml

```diff
@@ -0,0 +1,65 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: alertmanager
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        name: alertmanager
+    spec:
+      containers:
+      - name: alertmanager
+        image: prom/alertmanager:latest
+        args:
+          - "--config.file=/etc/alertmanager/config.yml"
+          - "--storage.path=/alertmanager"
+        ports:
+        - name: alertmanager
+          containerPort: 9093
+        volumeMounts:
+        - name: config-volume
+          mountPath: /etc/alertmanager
+        - name: alertmanager
+          mountPath: /alertmanager
+      volumes:
+      - name: config-volume
+        configMap:
+          name: alertmanager-config
+      - name: alertmanager
+        emptyDir: {}
```

Comment:
So the storage is not persistent. I know this is only an example, but is that what we want? Or will documentation make clear that users should adapt the example to use persistent storage?

Reply:
IMHO we do not want the users to use this for anything else than to test this. I'm sure there are many better ways how to setup Prometheus, Grafana or Alerting - these should be really just for testing purposes and we should make it clear in the docs. I think ideally we should get rid of this and integrate with the CoreOS Prometheus Operator instead.

Reply:
I agree that there are best ways.
The `prometheus-data` and `grafana-data` volume are the same; they are using `emptyDir` and up today no one (from the team) complained about that and it's not documented as well.
I will change/add the documentation accordingly.